### PR TITLE
Removing error codes from proto files

### PR DESCRIPTION
**Pull request by skisel-da**
These are be superseded by error codes which are self-documenting.

### CODE REVIEW

**Comment by skisel-da** [comment]
I have removed all the mentions of the gRPC error codes, should we be providing a link to all the error definitions somewhere? wdyt?

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto [hunk 1]
```diff
@@ -20,8 +20,6 @@ message Completion {
   string command_id = 1;
 
   // Identifies the exact type of the error.
-  // For example, malformed or double spend transactions will result in a ``INVALID_ARGUMENT`` status.
-  // Transactions with invalid time time windows (which may be valid at a later date) will result in an ``ABORTED`` error.
```
Comment (tudor-da) [inline ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:24]: I would redirect the user to the Ledger API error codes for reference and apply the same pattern for `InterfaceView.view_status`

**Comment by tudor-da** [comment]
Good stuff. Let's get the opinion of others as well since this impacts Ledger API client documentation. CC @bame-da @mziolekda

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto (hunk 1 above)
Comment (skisel-da) [reply ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:24]: changed

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto [hunk 2]
```diff
@@ -20,8 +20,7 @@ message Completion {
...
+  // The error is reported using the same rules for error codes and messages as the errors returned for API requests.
```
Comment (pbatko-da) [inline ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:23]:
```suggestion
  //  Uses the same format of conveying error details as is used for the LAPI rpcs.
```
?

**Review by pbatko-da — APPROVED** [review]

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto (hunk 2 above)
Comment (skisel-da) [reply ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:23]: There is currently zero occurrences of "LAPI" in the proto files. how about

```suggestion
  // Uses the same format of conveying error details as is used for the RPC responses of APIs.
```

Comment (mziolekda) [reply ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:23]: Yes, good point. I would also use a pronoun at the beginning of the sentence: "It uses..."

Comment (skisel-da) [reply ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto:23]:
```suggestion
  // It uses the same format of conveying error details as it is used for the RPC responses of the APIs.
```

**Pull request merged**